tools/SetupFlow/DevHome.SetupFlow/Services/WindowsPackageManager.cs

```diff
@@ -41,15 +43,18 @@ public class WindowsPackageManager : IWindowsPackageManager
     public WindowsPackageManager(
         WindowsPackageManagerFactory wingetFactory,
         IAppInstallManagerService appInstallManagerService,
-        IPackageDeploymentService packageDeploymentService)
+        IPackageDeploymentService packageDeploymentService,
+        IHost host)
     {
         _wingetFactory = wingetFactory;
         _appInstallManagerService = appInstallManagerService;
         _packageDeploymentService = packageDeploymentService;
 
+        var activityId = host.GetService<SetupFlowOrchestrator>().ActivityId;
```

Comment:
We should not depend on the SetupFlowOrchestrator inside this service

Reply:
Removed.